OWML.ModHelper.Menus/ModInputCombinationPopup.cs

```diff
@@ -16,7 +16,9 @@ public class ModInputCombinationPopup : PopupMenu
         public event Action OnPopupReset;
         public string Combination => string.Join("+", _combination.Select(ModInputLibrary.KeyCodeToString).ToArray());
 
+        [SerializeField]
         private SubmitAction _resetAction;
+        [SerializeField]
```

Comment:
What do these do?

Reply:
to make `Instantiate` work nicer

Reply:
😕 how?

Reply:
I don't think it would copy these things otherwise, as it not copies `Layout`

Reply:
I see 👍